app/app.go

```diff
@@ -768,9 +768,16 @@ func (app *App) AutoCliOpts() autocli.AppOptions {
 		}
 	}
 
+	moduleOptions := runtimeservices.ExtractAutoCLIOptions(app.ModuleManager.Modules)
+
+	// skip the comet consensus module autocli commands for v4.0.0
```

Comment:
```suggestion
	// Disable the comet consensus module autocli commands for v4.x.x because
	// there is a bug in the unmarshalling of blocks.
	//
	// https://github.com/celestiaorg/celestia-app/issues/4950
```